### PR TITLE
Add “pageview” event

### PR DESCRIPTION
**Pull request by lukeed**
- Moves `STARTED` to earlier in lifecycle
- Fires off "pageview" event for all posts, before API response
    - Includes `url` information for Dashboard bridge

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -513,6 +513,10 @@ export default function Vidy(opts) {
 
 		uri = `apps/${$.appid}/content/${$.postid}`; // base URI for payload; reused elsewhere
 
+		// What if I told you 😎
+		STARTED = Date.now(); // session has begun
```
Comment (eudisd) [inline src/index.js]: 👏

File: src/index.js [hunk 2]
```diff
@@ -513,6 +513,10 @@ export default function Vidy(opts) {
 
 		uri = `apps/${$.appid}/content/${$.postid}`; // base URI for payload; reused elsewhere
 
+		// What if I told you 😎
```
Comment (eudisd) [inline src/index.js]: What would you tell me? 😆

Comment (lukeed) [reply src/index.js]: the session has begun

Comment (eudisd) [reply src/index.js]: lol

**Pull request merged**